go/base_client.go

```diff
@@ -932,12 +932,9 @@ func (client *baseClient) MSetNX(ctx context.Context, keyValueMap map[string]str
 //
 // Note:
 //
-//	In cluster mode, if keys in `keys` map to different hash slots, the command
-//	will be split across these slots and executed separately for each. This means the command
-//	is atomic only at the slot level. If one or more slot-specific requests fail, the entire
-//	call will return the first encountered error, even though some requests may have succeeded
-//	while others did not. If this behavior impacts your application logic, consider splitting
-//	the request into sub-requests per slot to ensure atomicity.
+//	When in cluster mode, the command may route to multiple nodes when keys in `keys` map to different hash slots.
```

Comment:
I think we can add same for `DEL` and `UNLINK` commands because they never fail.